src/lib/user/user-models.ts

```diff
@@ -64,6 +64,23 @@ export type MyDogDetails = StatusSet & {
   dogReports: MyDogReport[];
 };
 
+/**
+ * The dog fields that can be updated prior to the first medical report. This
+ * can be used by registration and add-dog.
+ */
+export type MyDogRegistration = {
+  dogName: string;
+  dogBreed: string;
+  dogBirthday: Date;
+  dogGender: DogGender;
+  dogWeightKg: number | null;
+  dogDea1Point1: DogAntigenPresence;
+  dogEverPregnant: YesNoUnknown;
+  dogEverReceivedTransfusion: YesNoUnknown;
+  dogPreferredVetId: string;
+};
+
+// WIP: Remove MyDogRegistrationUpdate. Use MyDogRegistration instead.
```

Comment:
Change to TODO. Should be related to the ticket created for updateMyDogRegistration

Reply:
Changed 78ed087